src/styles/blocks/card.scss

```diff
@@ -0,0 +1,106 @@
+.card {
+  box-sizing: border-box;
+  width: $cardWidth;
+  padding: 32px 16px 16px;
+
+  margin-right: 48px;
+  margin-bottom: 46px;
+
```

Comment:
margin is an external style and should be inside elements not blocks. (to follow BEM)

Reply:
Yeah unfortunately this margin is because we cannot use gap for the flexbox.